app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,8 @@
 <resources>
     <string name="app_name">KotlinBaseProject</string>
-    <string name="welcome_text_field_text">Hola!</string>
-    <string name="check_button_text">OK</string>
+    <string name="parking_available_field_text">0</string>
```

Comment:
this zero should be in the integers resource

Reply:
this will be always a sting but the zero should be in an integer resource

Reply:
done

Reply:
this still here